e2e/testcafe-devextreme/tests/dataGrid/rowDragging.ts

```diff
@@ -744,3 +746,40 @@ test.skip('Item should appear in a correct spot when dragging to a different pag
     showBorders: true,
   });
 });
+
+// T1179218
+safeSizeTest('Rows should appear correctly during dragging when virtual scrolling is enabled and rowDragging.dropFeedbackMode = "push"', async (t) => {
+  const dataGrid = new DataGrid('#container');
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  // drag the row down
+  await dataGrid.moveRow(0, 30, 150, true);
+  await dataGrid.moveRow(0, 30, 350);
+
+  // waiting for autoscrolling
+  await t.wait(2000);
```

Comment:
Can we change it more stable
await t
    .expect(Selector("[aria-rowindex=30]").visible) // getRowByIndex
    .ok()